wot-wg-2023-draft.html

```diff
@@ -380,26 +380,25 @@ <h3>
               This will be a revision of the current <i>Web of Things (WoT) Discovery</i> specification published
               at <a href="https://www.w3.org/TR/wot-discovery/">https://www.w3.org/TR/wot-discovery/</a>,
               as a Candidate Recommendation on 19 January 2023.</p>
-          </dd>
-          <dt id="binding-deliverable" class="spec">Web of Things (WoT) Binding Templates</dt>
-          <dd>
-            <p>
-              This specification defines the mechanism to bind protocols, payload and media formats, or combination of
-              the two to the Web of Things.
-              The binding mechanism will be a normative document with the following set of registry sections:
-            <ul>
-              <li>3 registry sections dedicated to stable bindings that reach at least a working draft status</li>
-              <li>3 registry sections dedicated to experimental bindings that are at least publicly available</li>
-            </ul>
-            </p>
+            </dd>
+            <dt id="binding-deliverable" class="spec">Web of Things (WoT) Binding Templates</dt>
+            <dd>
+              <p>
+                This specification normatively defines and summarizes the structural mechanism to bind protocols, 
+                payload and media formats in order to allow integration of ecosystems into the Web of Things, using the already-existing
+                extension points of the Thing Description deliverable.
+                In addition to the core mechanism, this document specifies the policy and requirements in order to include 
```

Comment:
Not sure the policy for adoption should be defined IN the document, but the WG should decide upon a policy.  How about...
```suggestion
                The Working Group will define the policy and requirements in order to include 
```

Reply:
Ege has noted however that in order to use a registry, the policy does need to be included in the document.  Note that we don't SAY that we will use a registry here, but my proposed text still allows that, if that's what we decide to do.